test/functional/wallet_import_rescan.py

```diff
@@ -116,10 +116,19 @@ def setup_network(self):
                 extra_args[i] += ["-prune=1"]
 
         self.add_nodes(self.num_nodes, extra_args=extra_args)
+
+        # Import keys
```

Comment:
Can you add comment explaining need for this override? The stop_nodes immediately followed by start_nodes below seems very odd. And given that this is a wallet test, maybe it would be simpler to just keep the old behavior and just use the wallet instead of generating to hardcoded addresses?